pytorch_lightning/plugins/ddp_plugin.py

```diff
@@ -130,3 +131,23 @@ def get_model_from_plugin(
         if isinstance(model, LightningDistributedDataParallel):
             return model.module
         return model
+
+    def required_plugins(self, amp_backend: AMPType) -> Optional[list]:
```

Comment:
Let's move it to some common attribute class?
then no need to copy-paste everywhere...